changelogutils/changelog_test.go

```diff
@@ -18,13 +18,13 @@ import (
 	"github.com/spf13/afero"
 )
 
-var _ = Describe("ChangelogTest", func() {
+var _ = FDescribe("ChangelogTest", func() {
```

Comment:
Probably should add that "failOnFocus" flag or whatever it is.
```suggestion
var _ = Describe("ChangelogTest", func() {
```